<?xml version="1.0" ?>

<launch>
    <arg name="keyboard_node_name" value="keyaboard_move_node" />
    <arg name="publish_topic_right" value="rightCommand" />
    <arg name="publish_topic_left" value="leftCommand" />

    <node name="$(arg keyboard_node_name)" pkg="engrais_control" type="robot_keyboard_move" output="screen">
        <param name="pub_topic_right" type="string" value="$(arg publish_topic_right)"/>
        <param name="pub_topic_left" type="string" value="$(arg publish_topic_left)"/>
    </node>

</launch>